main.cpp

```diff
@@ -416,12 +511,26 @@ struct Oven
 {
     Oven();
     int numberBakingSheets, numberPrograms;
-    double highestTemp { 300.0 };
-    double currentTemp { 220.0 };
+    int highestTemp { 300 };
+    int currentTemp { 220 };
     std::string currentProgram;
     void bakeCake(std::string typeOfCake = "apple", int preHeat = 180);
     void bakePizza(std::string typeOfPizza = "margherita", int preHeat = 220);
     void makeRoast(std::string typeOfRoast = "beef", int setTemp = 230);
+    int adjustTemp(int targetTemp = 200, int current = 25)
+    {
+        current = currentTemp;
+        std::cout << "current: " << currentTemp << ", target is: " << targetTemp << std::endl;
+        int dir;
+        current<targetTemp ? dir = 1 : dir = -1;
```

Comment:
don't declare `dir` and not initialize it.
write this:
```
int dir = (expr ? a : b);
```
also put space around your `<` for improving the code readability.
- [ ] revise